File: Day 3/python/main.py
count = 0
while (True):
    print("\nEnter length of array. It cannot be less than 2.")
    count = int(input("Enter here: "))
    if count >= 2:
        break
    else:
        print("\nInvalid.")

nums = []

for i in range(1, count + 1):
    num = int(input(f"\nEnter integer {i}: "))
    nums.append(num)

good_pairs_count = 0

for i in range(len(nums)):
    for j in range(i + 1, len(nums)):
        if nums[i] == nums[j]:
            good_pairs_count += 1

print(f"Number of good pairs: {good_pairs_count}")

File: Day 3/kotlin/main.kt
import java.util.Scanner

fun main() {
    val nums = mutableListOf<Int>()

    val reader = Scanner(System.`in`)

    var count = 0;

    while (true) {
        println("Enter length of array. It cannot be less than 2.")
        count = reader.nextInt()
        if (count >= 2)
            break
        else
            println("Invalid.")
    }

    for (i in 1..count) {
        print("\nEnter integer $i: ")
        val num = reader.nextInt()
        nums.add(num)
    }

    var goodPairCount = 0

    for (i in 0 until count) {
        for (j in (i + 1) until count) {
            if (nums[i] == nums[j])
                goodPairCount++
        }
    }

    println(goodPairCount)
}